zerver/webhooks/jotform/doc.md

```diff
@@ -14,6 +18,18 @@ Get Zulip notifications for your Jotform responses!
 1. Add the above constructed **url** to the field, and click the
    **COMPLETE INTEGRATION** button.
 
```

Comment:
In Harbor, Greenhouse and Jotform docs we can use "generated" instead of "constructed" for the URL.